controller/script/simulate-proxy/main.go

```diff
@@ -202,12 +301,35 @@ func randomPod(pods []*v1.Pod, prvPodIp *common.IPAddress) *common.IPAddress {
 	return podIp
 }
 
+func randomPod(pods []*v1.Pod) *v1.Pod {
+	var pod *v1.Pod
+	for {
+		pod = pods[rand.Intn(len(pods))]
+		if strings.HasPrefix(pod.GetNamespace(), "kube-") {
+			continue // skip pods in the kube-* namespaces
+		} else {
+			break
+		}
+	}
+	return pod
+}
+
+func getRandomNamespaceKey(podsByNamespace map[string][]*v1.Pod) string {
+	choice := rand.Intn(len(podsByNamespace))
+	var chosen int
+	for k := range podsByNamespace {
+		if chosen == choice {
+			return k
+		}
+		chosen++
+	}
+	return ""
+}
+
 func main() {
 	rand.Seed(time.Now().UnixNano())
-
-	addr := flag.String("addr", ":8086", "address of proxy api")
-	requestCount := flag.Int("requests", 0, "number of api requests to make (default: infinite)")
 	sleep := flag.Duration("sleep", time.Second, "time to sleep between requests")
+	metricsAddrs := flag.String("metrics-addrs", ":9000,:9001,:9002", "range of network addresses to serve prometheus metrics")
```

Comment:
i imagine testing scenarios where we want to simulate 10 or 100 proxies. consider a range format for this parameter.

we don't necessarily need to do it for this branch, so ok to file it as an issue.